ffi_tests/tests/main.rs

```diff
@@ -81,3 +81,20 @@ fn test_csharp_code ()
             .success()
     );
 }
+
+#[test]
+fn test_lua_code()
+{
+    let output = ::std::process::Command::new("luajit")
```

Comment:
This adds a dependency on `luajit` to run tests locally. Worth mentioning in [README](https://github.com/getditto/safer_ffi#development).

Reply:
Added instructions to the README.